Trivago/Jade/Application/Framework/JadeBundle/Controller/JsonApiController.php

```diff
@@ -183,9 +196,17 @@ public function createEntityAction($resourceName, Request $httpRequest)
     }
 
     /**
-     * @param string $resourceName
+     * @param string  $resourceName
      * @param Request $httpRequest
+     *
      * @return JsonApiResponse
+     *
+     * @throws InvalidPath
+     * @throws AccessDeniedException
+     * @throws InvalidRequest
+     * @throws \LogicException
```

Comment:
Normally you only define exceptions that are meant to be captured specially for services.
I don't really see a point of defining this in the controllers
And it wasn't a controller \LogicException is not to be caught at all!